browser/js/app-initializer.js

```diff
@@ -149,7 +149,7 @@ export class AppInitializer {
 				}
 
 				if (this.enabledFeatures.feedback && flags.get('qualtrics')) {
-					feedback.init();
+					feedback.init(appInfo);
```

Comment:
does it need `appInfo`?

Reply:
Yeah, that's how we get the name of the app (e.g. article, stream, etc.).
I could scrape that info myself from the `<html>` tag, but `n-ui` does that already, so I thought it made sense to use it.